deployer/src/algolia_helper.py

```diff
@@ -68,14 +66,15 @@ def delete_docsearch_key(config):
 
 
 def delete_docsearch_index(config):
-    algolia_client_prod.delete_index(config)
+    algolia_index = algolia_client_prod.init_index(config)
+    algolia_index.delet()
 
 
 def list_index_analytics_key(config_name):
     analytics_keys = []
     keys = algolia_client_prod.list_api_keys()['keys']
     for key in keys:
         if 'indexes' in key and config_name in key[
-            'indexes'] and 'analytics' in key['acl']:
+                'indexes'] and 'analytics' in key['acl']:
```

Comment:
Isn't python going to throw an error because of the extra tab?

Reply:
I had no issue but I'll remove it just in case